packages/quantic/force-app/main/default/lwc/quanticFacetManager/e2e/pageObject.ts

```diff
@@ -0,0 +1,66 @@
+import type {Locator, Page} from '@playwright/test';
+import {
+  insightSearchRequestRegex,
+  searchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+
+export class FacetManagerObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get facetManager(): Locator {
+    return this.page.locator('c-quantic-facet-manager');
+  }
+
+  get facetManagerItems(): Locator {
+    return this.facetManager.locator('.facet-manager__item');
+  }
+
+  getFacetManagerItemByIndex(index: number): Locator {
+    return this.facetManagerItems.nth(index);
+  }
+
+  /**
+   * Filters out the date_input facets because they are not displayed in the facet manager.
+   */
+  filterInputFacets(facets: any): string[] {
+    return facets
+      .map((facet: any) => facet.facetId)
+      .filter((id: string) => !/_input$/.test(id));
+  }
+
+  async mockFacetOrder(facetIds: string[], useCase: string): Promise<void> {
```

Comment:
```suggestion
  async mockSearchFacetOrder(facetIds: string[], useCase: string): Promise<void> {
```

Reply:
this method  seems to belong more to the Search Object where we have already some methods that have the same pattern of applying some mocks to search requests, I think we should stay consistent with our pattern and keep it there.
Additionally the SearchObject accepts a `searchRequestRegex` as a parameter so you would setup evrything properly in your fixture and you will no longer need the logic about 'if insight use this regex if not use this other regex..'

Reply:
good idea!